### DISCUSSION TITLE
How to remove trailing mask characters on backspace

### DISCUSSION THREAD

**Discussion by mattstobbs** [uNmAnNeR/imaskjs#703] [Q&A] [answered]
Hey, I'm using a simple mask of `'0000 0000'`. It works great when typing numbers but when the backspace is clicked, if there are only four numbers left, there is a trailing space. For example
```
"1234"
{Click 5}
"1234 5"
{Backspace}
"1234 "
```
Is there a way to remove the trailing space on backspace?

**Comment by uNmAnNeR** [comment]
@mattstobbs  `eager: true` ?

**Reply by mattstobbs** [reply]
Thanks for the response! That removes the trailing space on backspace but it also adds a space when I enter the fourth character. I want there to never be a trailing space if there is only 4 characters.

**Reply by uNmAnNeR** [reply]
yeah i was thinking about adding `eager: 'remove'` or `eager: 'insert'` for such cases so may be later

**Reply by uNmAnNeR** [reply, marked as answer]
added
